### DISCUSSION TITLE
no common history

### DISCUSSION THREAD

**Discussion by aboulkassm** [community/community#56159] [New to GitHub]
### Select Topic Area

Question

### Body

its my first time using GitHub ,, I am working on a project with five Othrers, now trying to merge master branch with another branch and couidnt make it ,,this is how the history looks ,,
* 0c0c1b0 (remote) notification & homepage
* 229ac6f Dashboard layout
* ae5a969 (origin/master) Merge remote-tracking branch 'main/master'
*   3f022a9 Merge remote-tracking branch 'main/master'
|\
| *   05839dd Merge remote-tracking branch 'main/master'
| |\
* | \   b191988 Merge remote-tracking branch 'main/master'
|\ \ \
| |/ /
|/| /
| |/
| * 6255e0d Merge remote-tracking branch 'main/master'
* | 15c6142 Merge remote-tracking branch 'main/master'
|/
* 4582083 Merge remote-tracking branch 'main/master'
* 0ad7dcf homepage-Notification
* eb70abf homepage-Notification
* aaf8978 added Module for wear os and default activity
* 8616bc8 notification-manager
* 227de3a DB-sqlLite
* f892700 app
* faccee3 Completed sensor data collection and digit span task scheduling
*   8792f3d Merge remote-tracking branch 'main/master'
|\
| *   483785f Merge remote-tracking branch 'main/master'
| |\
* | \   94f36b0 Merge remote-tracking branch 'main/master'
|\ \ \
| |/ /
|/| /
| |/
| * 360b419 Merge remote-tracking branch 'main/master'
* | 51ee535 Merge remote-tracking branch 'main/master'
|/
*   bee64ac Merge remote-tracking branch 'main/master'
|\
| *   bf94dcc Merge remote-tracking branch 'main/master'
| |\
* | \   120f298 Merge remote-tracking branch 'main/master'
|\ \ \
| |/ /
|/| /
| |/
| * 234dd4c homepage-Notification
* | 29f25cb homepage-Notification
|/
*   bb47b27 Merge remote-tracking branch 'main/wearOs'
|\
| * 311bcfe (origin/wearOs) added Module for wear os and default activity
* | 4ed6b8c notification-manager
* | cdc670d notification-manager
|/
* b6a6075 DB-sqlLite
* efdc610 app
. need help plz

**Comment by queenofcorgis** [comment]
Hey there! I'm sure our community would love to help, but this is just the history from the terminal, it would be helpful to know why you're having trouble merging. What commands are you running and what error messages are you getting back?
